src/actions/receive-profile.js

```diff
@@ -345,10 +350,29 @@ export function finalizeFullProfileView(
       timelineType = determineTimelineType(profile);
     }
 
+    let selectedTab = getSelectedTab(getState());
```

Comment:
nit: Can we add a short code comment here maybe telling them that we are doing this for profiles without samples data.